client/app.js

```diff
@@ -9,10 +9,11 @@ import store from './store'
 
 import { isLoaded } from './reducers'
 import { Stories, Story, CreateStory } from './story'
+import { Groups, Group } from './group'
```

Comment:
We don't have a page for a single Group (and I'm pretty sure the table of all groups is likely good enough) so I think we can get rid of it.